proto/config.proto

```diff
@@ -15,9 +15,11 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package proto;
+syntax = "proto2";
+package cockroach.proto;
+option go_package = "proto";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "gogoproto/gogo.proto";
```

Comment:
Does this help anything? Just curious about the motivation behind the `PROTO_PATH` changes.